src/CMakeLists.txt

```diff
@@ -104,6 +103,11 @@ add_library(migraphx
     value.cpp
     verify_args.cpp
 )
+if(WIN32)
+    target_sources(migraphx PRIVATE process_win32.cpp)
```

Comment:
Dont create another source file, just add an ifdef to the process.cpp file.

Reply:
Ok, I will move it to the `process.cpp`. I thought you asked me to do that in the Draft PR, but I might mistaken that with someone else asking me from a different component.